lib/PDF/API2/Annotation.pm

```diff
@@ -193,12 +195,17 @@ sub pdf {
     $self->{'A'}->{'F'} = PDFStr($file);
 
     unless (%options) {
-        my $destination = PDFNum($page_number);
-        $self->{'A'}->{'D'} = _destination($destination, $location, @args);
+	if ( $dest =~ /^#(.+)/ ) { # named dest
+	    $self->{'A'}->{'D'} = PDFStr($1);
```

Comment:
If I'm reading the spec correctly, should PDFStr be replaced with PDFName?  The type for D on GoToR actions is listed as "name, byte string, or array".

Reply:
I tend to agree, but what would then be the reason the spec includes (byte) string?
A name object can contain an arbitrary sequence of characters, just like strings. Restricting to name would not impose any semantic difference.

Reply:
Changed.